### PR TITLE
Album routes

### PR DESCRIPTION
**Pull request by rrodrig003**

### CODE REVIEW

File: server/api_routes/albumRoutes.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+      console.error(chalk.red('Failed to find any albums', e));
+      next(e);
+    });
+});
+
+// GET A SINGLE ALBUM BY ID FOR A GIVEN USER
+router.get('/:userId/:albumId', (req, res, next) => {
+  User.findOne({
+    where: {
+      id: req.params.participantId,
```
Comment (Tsunenori-L) [inline server/api_routes/albumRoutes.js]: Do you mean userId?

File: server/api_routes/albumRoutes.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    newAlbum.setUsers(owner);
+    console.log(chalk.green('Successfully CREATED album'));
+  } catch (e) {
+    console.error(chalk.red('Failed to post new album', e));
+    next(e);
+  }
+});
+
+// PUT (UPDATE) ALBUM WITH IMAGES
+router.put('/addImages', async (req, res, next) => {
```
Comment (dlevene) [inline server/api_routes/albumRoutes.js]: this should have /:albumId

File: server/api_routes/albumRoutes.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    // participant association
+    newAlbum.setUsers(owner);
+    console.log(chalk.green('Successfully CREATED album'));
+  } catch (e) {
+    console.error(chalk.red('Failed to post new album', e));
+    next(e);
+  }
+});
+
+// PUT (UPDATE) ALBUM WITH IMAGES
```
Comment (dlevene) [inline server/api_routes/albumRoutes.js]: @rrodrig003 
We will also need a PUT to update album with more participants

File: server/api_routes/albumRoutes.js (hunk 1 above)
Comment (rrodrig003) [reply server/api_routes/albumRoutes.js]: Yes, thanks! made change.

File: server/api_routes/albumRoutes.js (hunk 2 above)
Comment (rrodrig003) [reply server/api_routes/albumRoutes.js]: Yup. moved from body to URL.

File: server/api_routes/albumRoutes.js (hunk 3 above)
Comment (rrodrig003) [reply server/api_routes/albumRoutes.js]: added

**Pull request merged**